### DISCUSSION TITLE
Selecting an item from list in Select component is deselecting marked text.

### DISCUSSION THREAD

**Discussion by or313** [mantinedev/mantine#3617] [Q&A]
Hey everyone, I'm having an issue which starts with a highlighted text which is outside of the boundaries of my application. I'm using the Select component and when I click on one of the options in the dropdown, the focus is being taken away from my highlighted text and put on the input element. Does anyone have any ideas on how to prevent this from happening? Any help would be greatly appreciated. Thanks!

**Comment by rtivital** [comment]
It is how Select (and all other inputs) works. If Select is not focused then it is an accessibility issue.

**Reply by or313** [reply]
Yes i understand but still can you think of a way to overrule this behavior?

**Reply by rtivital** [reply]
I do not think that there is a good way of doing so. You can explore component source code to find out about possible ways.
